http_client.go

```
@@ -27,8 +21,8 @@ func postRequest(url string, body []byte) []byte {
 	return data
 }
 
-func (r getResponse) getRequest(url string) []byte {
-	req, _ := http.NewRequest("GET", url, nil)
+func getRequest(url string) []byte {
+	req, _ := http.NewRequest("GET", baseURL+"/"+url, nil)
```

we shouldn't ignore this error.